src/plugins/janus_streaming.c

```diff
@@ -478,6 +479,16 @@ multistream-test: {
 	"id" : <unique ID of the just edited mountpoint>,
 	"permanent" : <true|false, depending on whether the changes were saved to configuration file or not>
 }
+\endverbatim
+ *
+ * A successful \c edit will also result in an \c edited event sent to all viewers when the metadata has changed:
```

Comment:
This is missing a prefix like

> In case \c edited_event was set to \c true in the request, a successful...

As it is right now, the text suggests that the only required precondition is metadata being edited.

Reply:
I have updated the text.